petab_select/candidate_space.py

```diff
@@ -585,6 +1307,13 @@ def is_plausible(self, model: Model) -> bool:
             return True
         return False
 
+    # TODO does Lateral need this?
```

Comment:
check

Reply:
So as much as I know, no. Because Forward uses it only for _consider_method, and in Lateral _consider_method doesn't need distance. 

But I don't know if distance is called anywhere else.

Reply:
Thanks, then I'll remove it and see if something breaks :sweat_smile: